qiskit_nature/results/utils/protein_shape_file_gen.py

```diff
@@ -128,10 +128,12 @@ def save_xyz_file(
                 be saved in the current working directory.
             comment: Comment to be added to the second line of the file. By default the line will
                 be left blank.
-            overwrite: If True, the file will be overwritten if it already exists.
+            overwrite: If ``True``, the file will be overwritten if it already exists.
+        Raises:
+            FileExistsError: If the file already exists and overwrite is False.
         """
         file_path = os.path.join(path, name + ".xyz")
-        if os.path.exists(file_path):
+        if os.path.exists(file_path) and not overwrite:
```

Comment:
(for some reason I cannot add a suggestion directly via Github right now...)
If you write the following instead:
```python
if not overwrite and os.path.exists(file_path):
```
you can avoid the comparatively costly file system check in the default case where `overwrite=False`.